### DISCUSSION TITLE
Windows 10 start menu turn off theme-aware tiles

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1729] [Ideas]
I was thinking of adding to theme-aware tiles turn on (default) or turn off in Windows 10 start menu.

If it is not planned yet, I hope this will be a thing in the next update.

![Start-menu-comparison](https://github.com/valinet/ExplorerPatcher/assets/140480110/a3e8253b-cdda-46bc-b853-bdbd6054ca0f)

The code is here:

<?xml version="1.0" encoding="utf-8"?>
<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:wincap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/windowscapabilities">
	<Identity Name="Microsoft.Windows.StartMenuExperienceHost" Publisher="CN=Microsoft Windows, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="10.0.19041.1" ResourceId="neutral"/>
	<Properties>
		<DisplayName>ms-resource:StartMenuExperienceHost/PkgDisplayName</DisplayName>
		<PublisherDisplayName>ms-resource:StartMenuExperienceHost/PublisherDisplayName</PublisherDisplayName>
		<Logo>Assets\StoreLogo.png</Logo>
	</Properties>
	<Resources>
		<Resource Language="en-us"/>
	</Resources>
	<Dependencies>
		<TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.19041.1"/>
	</Dependencies>
	<Applications>
		<Application Id="App" Executable="StartMenuExperienceHost.exe" EntryPoint="StartMenuExperienceHost.App">
			<uap:VisualElements DisplayName="ms-resource:StartMenuExperienceHost/AppName/Text" Square150x150Logo="Assets\Logo.png" Square44x44Logo="Assets\SmallLogo.png" Description="ms-resource:StartMenuExperienceHost/AppDescription" BackgroundColor="#001020" AppListEntry="none">
			</uap:VisualElements>
			<uap:ApplicationContentUriRules>
				
				<uap:Rule Match="http://microsoft.com/thisisasitethatdoesntexist" Type="include" WindowsRuntimeAccess="none"/>
			</uap:ApplicationContentUriRules>
			<Extensions>
				
				<wincap3:Extension xmlns:wincap3="http://schemas.microsoft.com/appx/manifest/foundation/windows10/windowscapabilities/3" Category="windows.internal.componentUI"/>
				<Extension  Category="windows.backgroundTasks" EntryPoint="WindowsInternal.Shell.UnifiedTile.CuratedTileCollections.CollectionInitializationBackgroundTask">
					<BackgroundTasks>
						<Task Type="systemEvent"/>
						
						<Task Type="timer"/>
					</BackgroundTasks>
				</Extension>
			</Extensions>
		</Application>
	</Applications>
	<Capabilities>
		<wincap:Capability Name="shellexperience"/>
		
		<Capability  Name="internetClient"/>
		<rescap:Capability xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" Name="packageQuery"/>
		<wincap:Capability Name="packageContents"/>
		<wincap:Capability Name="visualElementsSystem"/>
		<wincap:Capability Name="userAccountInformation"/>
		<rescap:Capability xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" Name="activitySystem"/>
		<wincap:Capability Name="activityData"/>
		<Capability  Name="privateNetworkClientServer"/>
		<rescap:Capability xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" Name="previewStore"/>
		<wincap:Capability Name="cloudStore"/>
		<wincap:Capability Name="targetedContent"/>
		<wincap:Capability Name="storeAppInstall"/>
		<wincap:Capability Name="storeLicenseManagement"/>
		<uap:Capability Name="contacts"/>
		<wincap:Capability Name="cortanaSettings"/>
		<rescap:Capability xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" Name="cortanaPermissions"/>
	</Capabilities>
	<Extensions>
		
		<Extension  Category="windows.activatableClass.inProcessServer">
			<InProcessServer>
				
				<Path>JumpViewUI.dll</Path>
				
				<ActivatableClass ActivatableClassId="JumpViewUI.jumpviewui_XamlTypeInfo.XamlMetaDataProvider" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.App" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.FlyoutControlHost" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.FocusHelper" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.JumpListControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.JumpListListView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.TaskbarJumpListFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.ToolTipBehavior" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.ToolTipViewModel" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewActionControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewHeaderControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewItemControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewListView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.JumpViewMetrics" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="JumpViewUI.Legacy.OfferDeleteFlyoutControl" ThreadingModel="both"/>
			</InProcessServer>
		</Extension>
		<Extension  Category="windows.activatableClass.inProcessServer">
			<InProcessServer>
				<Path>StartUI.dll</Path>
				
				<ActivatableClass ActivatableClassId="StartUI.AllAppsFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.AllAppsObjectModelFactory" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.AllAppsPane" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.App" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.BadgeDataTemplateSelector" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.DesktopAppActivator" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.Events" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.FrameMetrics" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.GridLayoutMetrics" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.GridMetrics" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.GroupHeaderControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.HostSpecificTestHooks" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.JumpViewControlHost" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.MobileFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.NavigationPaneView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.NewTabPageObjectModel" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.PowerOptionsView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.SplitViewFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.StartModel" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.StartProperties" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.startui_XamlTypeInfo.XamlMetaDataProvider" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.StartContext" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.ThemeManager" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileAppspaceDataTemplateSelector" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileGridHost" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileGridItemTemplateSelector" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileGridView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileGridXamlUtils" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileGroupViewControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileListView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileListViewItem" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileMetrics" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TilesFrame" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.TileViewControl" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.UserTileView" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.VisualCacheManagerHost" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.ManageModeEventPayload" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.ViewFactory" ThreadingModel="both"/>
				<ActivatableClass ActivatableClassId="StartUI.Win32HostedExperience" ThreadingModel="both"/>
			</InProcessServer>
		</Extension>
	</Extensions>
</Package>

**Comment by XuanChen1234** [comment]
@valinet Any plans for the Windows 10 theme-aware tiles on and off?

**Comment by gricardo93** [comment]
Hello. Do you have any update on this? It would be perfect to be able to use it on Windows 11 too
